apis/mattermost/v1beta1/mattermost_types.go

```diff
@@ -147,6 +152,11 @@ type Ingress struct {
 	// Host defines the Ingress host to be used when creating the ingress rules.
 	// +optional
 	Host string `json:"host,omitempty"`
+
+	// Hosts allows specifying additional domain names for Mattermost to use.
+	// +optional
+	Hosts []IngressHost `json:"rules,omitempty"`
```

Comment:
Is `json:"rules,omitempty"` what you were meaning to do here?

Reply:
Fixed, just renaming artifact